cfme/tests/services/test_myservice.py

```diff
@@ -388,3 +388,86 @@ def test_load_service_with_button(request, appliance, generic_service, import_di
         custom_button_group = Dropdown(view, button_gp.text)
         assert custom_button_group.is_displayed
         custom_button_group.item_select(button.text)
+
+
+@pytest.mark.meta(automate=[1660637])
+@pytest.mark.customer_scenario
+def test_retire_service_bundle_and_vms(appliance, provider, catalog_item, request):
+    """
+    Bugzilla:
+        1660637
+
+    Polarion:
+        assignee: nansari
+        startsin: 5.10
+        casecomponent: Services
+        initialEstimate: 1/6h
+        testSteps:
+            1. Create service catalog item
+            2. Create bundle with above catalog item
+            3. Order the Service bundle
+            4. Navigate to my service
+            5. Retire the service
+        expectedResults:
+            1.
+            2.
+            3.
+            4.
+            5. the service should be retired and VM should be retired and archived
+    """
+    collection = provider.appliance.provider_based_collection(provider)
+    vm_name = "{}0001".format(catalog_item.prov_data["catalog"]["vm_name"])
```

Comment:
Please consider using f-strings consistently for future contributions - there's a mix here, but we're not enforcing its use at this time, so I'm not going to block the merge.